zebra/zebra_nhg.c

```diff
@@ -3385,13 +3385,7 @@ struct nhg_hash_entry *zebra_nhg_proto_add(struct nhg_hash_entry *nhe,
 	session = nhe->zapi_session;
 	instance = nhe->zapi_instance;
 
-	if (CHECK_FLAG(nhg->flags, NEXTHOP_GROUP_TYPE_GROUP)) {
-		if (IS_ZEBRA_DEBUG_NHG)
-			zlog_debug("%s: id %u, nexthop group ids not supported",
-				   __func__, id);
-		return NULL;
-	}
-	if (!nhg->nexthop) {
+	if (!CHECK_FLAG(nhg->flags, NEXTHOP_GROUP_TYPE_GROUP) && !nhg->nexthop) {
```

Comment:
I'm not sure I agree with this change.  What is the goal here?  Why is it important to loose the check for a nexthop group type with it's own error message.

Reply:
this was temp message to avoid crashing if someone attempts to compile and run previous commit.